middlewares/authentication.js

```diff
@@ -10,7 +10,7 @@ const verifyToken = (req, res, next) => {
     let error;
     if (!token) {
       error = new Error('No token provided');
-      error.status = 400;
+      error.status = 401;
```

Comment:
This is good. Should be `401`, not the `400` we initially put there.

Reply:
Thanks